src/main/kotlin/net/ccbluex/liquidbounce/event/events/PlayerEvents.kt

```diff
@@ -79,6 +80,18 @@ class PlayerUseMultiplier(var forward: Float, var sideways: Float) : Event()
 @Nameable("playerInteractedItem")
 class PlayerInteractedItem(val player: PlayerEntity, val hand: Hand, val actionResult: ActionResult) : Event()
 
+@Nameable("playerArmorInventory")
+@WebSocketEvent
+class PlayerArmorInventory(val stacks: List<ItemStack>) : Event()
+
+@Nameable("playerMainInventory")
+@WebSocketEvent
+class PlayerMainInventory(val stacks: List<ItemStack>) : Event()
+
+@Nameable("playerCraftingInventory")
+@WebSocketEvent
+class PlayerCraftingInventory(val stacks: List<ItemStack>) : Event()
```

Comment:
Make this one event and call it like `WsPlayerInventoryUpdateEvent`. No need to split it into 3 events ;)